android-components/components/ui/icons/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<resources>
+    <color name="mozac_ic_star_fill_primary">#CFCFD8</color>
```

Comment:
```suggestion
    <color name="mozac_ic_star_one_half_fill_primary">#CFCFD8</color>
```

Reply:
updated them to "mozac_ic_star_unfilled" and "mozac_ic_star_filled" since they're being used for the solid stars as well.